corehq/apps/app_manager/views.py

```diff
@@ -1272,10 +1273,32 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
         'sessionid': request.COOKIES.get('sessionid'),
         'features': vellum_features,
         'plugins': vellum_plugins,
+        'app_callout_templates': next(app_callout_templates),
     })
     return render(request, 'app_manager/form_designer.html', context)
 
 
+def app_callout_templates():
+    """Load app callout templates from config file on disk
+
+    Generator function defers file access until needed, acts like a
+    constant thereafter.
```

Comment:
cool pattern - never seen it before. 

is it right that the benefits of this are the lazy evaluation and fast / constant memory access, and the drawbacks are that if anyone ever tries to iterate over the whole thing they will create an infinite loop?

based on the failure mode I might advocate marking it "private" to the module with a leading `_`